packages/ui/src/views/repo/pull-request/details/components/conversation/pull-request-overview.tsx

```diff
@@ -309,15 +307,17 @@ const PullRequestOverview: React.FC<PullRequestOverviewProps> = ({
                         </div>
                       ),
                       selectStatus: (
-                        <PullRequestStatusSelect
-                          refetchActivities={refetchActivities}
-                          commentStatusPullReq={commentStatusPullReq}
-                          comment={{
-                            commentItems: commentItems
-                          }}
-                          pullReqMetadata={pullReqMetadata}
-                          repoId={repoId}
-                        />
+                        <div className="flex gap-x-2" key={`${index}-${payload?.id}`}>
```

Comment:
Using index in key is not a good idea, may be we can use index only if payload id is null. Something like key={payload?.id || index} ?

Reply:
`payload.id` should be sufficient if it's unique.
if it's empty, we should not be rendering it at all, right?

Reply:
done

Reply:
correct we shud not be rendering it